grails-app/domain/org/pih/warehouse/invoice/InvoiceItem.groovy

```diff
@@ -188,11 +203,14 @@ class InvoiceItem implements Serializable {
                 uom: unitOfMeasure,
                 amount: amount,
                 unitPrice: unitPrice,
-                totalAmount: totalAmount,
-                totalPrepaymentAmount: totalPrepaymentAmount,
                 orderAdjustment: orderAdjustment,
                 productName: product?.name,
-                displayNames: product?.displayNames
+                displayNames: product?.displayNames,
+                type: invoiceItemType?.name(),
```

Comment:
if you're calling name() here why do we need the type field in the enum?

Reply:
It is built on the `Enum`

Reply:
yes but I mean is InvoiceItemType.type being used anywhere? name() just uses the actual name of the enum itself.

Personally I prefer to define the toString() method of enums and use that (and in this case you probably want to return the type field). It lets us be more intentional about how our items get stringified.